### ISSUE TITLE
Failed to fetch current file. message = Bad credentials.

### ISSUE THREAD

**Issue by hrisha-ke** [ketsushiri/vk-scripts#1]
Проблема с токеном гитхаба. Фикс -- использовать свой. Нет гарантий, что текущий будет работать, хотя должен бы.

**Comment by ketsushiri** [comment]
Да, к слову, это не переизобретённый git, если что. Эта штука для других целей.
